hack/deploy_cincinnati.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# The script serves as an example of how Cincinnati can be run on an OpenShift cluster. 
+
+# Create a new namespace/project for Cincinnati
+oc new-project cincinnati
+
+# Create a dummy secret as a workaround to not having real secrets
+oc create secret generic cincinnati-credentials --from-literal=""
+
+# Apply oc template
+oc new-app -f dist/openshift/cincinnati.yaml \
+  -p IMAGE_TAG="90efacd" \
+  -p GB_PAUSE_SECS=300 \
+  -p GB_PLUGIN_SETTINGS="$(cat <<-EOF
+      [[plugin_settings]]
+      name = "release-scrape-dockerv2"
+      repository = "openshift-release-dev/ocp-release"
+      fetch_concurrency = 16
```

Comment:
do we have a reasonable default for this?  I find it unlikely that most users will feel a need to tune fetch concurrency.

Reply:
same as https://github.com/openshift/cincinnati/pull/260/files#r411293824